pkg/cli/context.go

```diff
@@ -24,7 +24,6 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/cli/clisqlexec"
 	"github.com/cockroachdb/cockroach/pkg/cli/clisqlshell"
 	"github.com/cockroachdb/cockroach/pkg/cli/democluster"
-	"github.com/cockroachdb/cockroach/pkg/clusterversion"
```

Comment:
Looks like this breaks the build.

Reply:
yeah, the roachtest nightlies failed to build,

- [GCE](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/14807967?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true&expandBuildProblemsSection=true)
- [AWS](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAwsBazel/14807968?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildProblemsSection=true&expandBuildChangesSection=true)

Reply:
doh, sorry about that! 

thank you, David, for fixing the merge skew https://github.com/cockroachdb/cockroach/pull/122275